### PR TITLE
getprioritizedhashfunction is not exposed to webapps

### LINKED ISSUE #346: [SRI] getPrioritizedHashFunction should be a normative definition
Section 3.2.2 states that a user agent MUST provide a means of determining algorithm priority, but doesn't normatively specify an API, only gives the example of "getPrioritizedHashFunction(a, b)".  If we are going to be able to test this requirement as a MUST, it needs to be normatively defined with a way to access it.

### PR DESCRIPTION
**Pull request by devd**
fixes #346

### CODE REVIEW

File: specs/subresourceintegrity/spec.markdown [hunk 1]
```diff
@@ -285,6 +285,9 @@ collision-resistant.  For example, `getPrioritizedHashFunction('sha256',
 'sha512')` would return `'sha512'` and `getPrioritizedHashFunction('sha256',
 'sha256')` would return the empty string.
 
+The `getPrioritizedHashFunction` implementation is vendor specific, and the interface
```
Comment (fmarier) [inline specs/subresourceintegrity/spec.markdown]: The verb tense here doesn't feel quite right. The previous paragraph says "if a user agent implemented getPrioritizedHashFunction() then it would ..." and in this change, it feels like we assume that it exists.

I suggest rewording to something like "Note that getPrioritizedHashFunction() is never exposed to applications given that both its implementation and its existence are vendor-specific."

**Comment by mozfreddyb** [comment]
Oops, my merge mid-aired with @fmarier's comment. Sorry.

**Comment by mozfreddyb** [comment]
I have reverted this in #377 and was hoping this would allow me to reopen here, but it doesnt :disappointed:
